fineract-provider/src/main/java/org/apache/fineract/useradministration/service/RoleWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -103,22 +103,20 @@ private void handleDataIntegrityIssues(final JsonCommand command, final Throwabl
     public CommandProcessingResult updateRole(final Long roleId, final JsonCommand command) {
         try {
             this.context.authenticatedUser();
-
```

Comment:
Check that your computer is not formatting whitespaces..

Reply:
This was my update, I remove it intentionally